utils/utils.py

```diff
@@ -78,8 +78,14 @@ def parse_date(
 	}
 	# parse the date with dateparser
 	date = dateparser.parse(date_str, settings=settings)
-	# make times PM if time is early in the day, base is PM, and "AM" is not specified
-	if date and date.hour < 8 and base.hour >= 12 and not "am" in date_str.lower():
+	# make times PM if time is early in the day, base is PM, and no indication that AM was specified
+	if (
+		date
+		and date.hour < 8
+		and base.hour >= 12
+		and not "am" in date_str.lower()
+		and not "T" in date_str
```

Comment:
whats this for?

Reply:
@abrahammurciano 

Google calendar returns dates like `2021-04-16T07:00:00+03:00` in iso format.

When getting the endpoints from the event, the date parser is used because it makes converting timezones easier.

Even though "AM" is not specified in this format, it is very clear since it's always 24-hour time.